src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -213,12 +213,25 @@ const constructPageFetchUrl = ({
 
     switch (pageType) {
       case ARTICLE_PAGE: {
-        const isOptimoId = isOptimoIdCheck(`/articles/${id}`);
+        const { assetId } = parseRoute(pathname);
+        const pathnameNoRendererExtension = removeRendererExtension(pathname);
```

Comment:
Is this extension removal required? Seems to work ok if I just use `pathname` for the `isOptimoIdCheck` and `isCpsIdCheck` functions.

Reply:
Is it worth just using the `assetId` that is returned from `parseRoute`? That should strip out just the ID part and should satisfy the Optimo/CPS ID checks.

Reply:
`parseRoute` also returns a `platform` value which can be `cps | articles | tipo`. We use it for the av-embeds amp/embed urls: https://github.com/bbc/simorgh/blob/4026d9acc621960027df0fe618962e53622e14aa/src/app/components/MediaLoader/utils/urlConstructors.ts#L23-L29

Should have probably called articles 'optimo' instead to be more accurate.

Just another option here, but no big deal if you want to keep it as-is.